src/components/ActionPage/Hero.tsx

```diff
@@ -43,15 +43,19 @@ const HeroDotsContainer = styled.div`
   width: 100vw;
 `;
 
-const HeroRow = styled.div`
+type HeroStyleProps = {
+  justifyContent?: string;
+};
+const HeroRow = styled.div<HeroStyleProps>`
   display: flex;
   width: 100%;
   flex-direction: column;
 
   @media (min-width: 768px) {
     text-align: left;
     flex-direction: row;
-    justify-content: space-between;
+    justify-content: ${(props) =>
+      props?.justifyContent ? props.justifyContent : "space-between"};
```

Comment:
```suggestion
      props?.justifyContent ?? "space-between"};
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Also, is `props` ever not an object? Why the `?`?

Reply:
Making the change and will re-submit. No question props is always an object, but it's property is not guaranteed to be there and my understanding is that props.justifyContent would throw an error when the property justifyContent isn't included.

Reply:
> props.justifyContent would throw an error when the property justifyContent isn't included

That's not the typical behavior. If you access a non-existing field, you get back `undefined`. There might be some shenanigans going on with `props`, but I don't think that's the case.

```node
> const a = {k: 1};
undefined
> a.k
1
> a.no
undefined
```

Since there's no good reason to have the `?`, let's take it out!

```suggestion
      props.justifyContent ?? "space-between"};
```

Reply:
fixed